packages/chrome-extension/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@clerk/chrome-extension",
+  "version": "0.1.0",
+  "types": "./dist/types/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./dist/esm/index.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsup --env.NODE_ENV production",
+    "dev": "tsup --watch",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@clerk/clerk-js": "^4.34.0-staging.0",
+    "@clerk/clerk-react": "^4.14.0-staging.0"
+  },
+  "devDependencies": {
+    "@testing-library/dom": "^8.19.0",
```

Comment:
We should probably move these deps to the root package.json as well. (Not as part of this PR)

Reply:
I've created an internal ticket.